lib/peanut.dart

```diff
@@ -57,12 +67,137 @@ Future<Null> run(String targetDir, String targetBranch, String commitMessage,
       print('There was no change in branch. No commit created.');
     } else {
       print('Branch "$targetBranch" was updated '
-          'with "pub build" output from "$targetDir".');
+          'with `$command` output from `$targetDir`.');
     }
   } finally {
     await tempDir.delete(recursive: true);
   }
 }
 
-Stream<String> getStrings(Stream<List<int>> std) =>
-    const LineSplitter().bind(SYSTEM_ENCODING.decoder.bind(std));
+Future<String> _runBuild(
+    String tempDir, String targetDir, String config) async {
+  var args = [
+    'run',
+    'build_runner',
+    'build',
+    '--assume-tty',
```

Comment:
Ha, I didn't know there is still a use for this. I was tempted to remove it once we got the INHERIT_STDIO stuff and stopped using this internally.

Reply:
Should I do something differently in how I'm calling here?

Reply:
I think it's fine. Just something we'll have to keep in mind that we might not be able to drop that flag.

Reply:
Ah – if I ran `pub run...` with `mode: ...INHERIT_STDIO` I should be cool, right?

Reply:
yup, then you could drop this flag and you'd get other benefits like ctrl-c working :)

Reply:
https://github.com/kevmoo/peanut.dart/issues/15